rules/group.go

```diff
@@ -230,7 +230,9 @@ func (g *Group) run(ctx context.Context) {
 			g.evalIterationFunc(ctx, g, evalTimestamp)
 		}
 
-		g.RestoreForState(time.Now())
+		now := time.Now()
```

Comment:
Nitpick, but perhaps calling it something like restpreStartTime may be more clear. It's no longer "now" after restoration finishes

Reply:
Fixed in https://github.com/prometheus/prometheus/pull/13974/commits/381a77ac1e1ef5e616ad45630dc6700a1916ba1d